Source/Cells/Popover/FORMSelectFieldCell.m

```diff
@@ -62,6 +62,13 @@ - (void)updateWithField:(FORMField *)field {
     } else {
         self.fieldValueLabel.text = nil;
     }
+    
+    if ([field.accessibilityLabel length] > 0) {
+        self.fieldValueLabel.accessibilityLabel = field.accessibilityLabel;
+    } else {
+        self.fieldValueLabel.accessibilityLabel = field.title;
+    }
+    self.fieldValueLabel.accessibilityValue = self.fieldValueLabel.text;
```

Comment:
The if statement is setting the accessibility **label** (`accessibilityLabel`), the line outside of that is setting the accessibility **value** (`accessibilityValue`).

If the `accessibility_label` JSON value is blank or not defined, it will use the `field.title` value otherwise it will use the defined label. After that, it will set the `accessibilityValue` property to the selected value as it is printed in the field.

**Assuming:**
`accessibility_label` = Visit time
`self.fieldValueLabel.text` = 6:00 PM

**The results is this voiceover audio:**
`Visit time six o'clock pm`